.gitignore

```diff
@@ -264,4 +264,5 @@ kubeconfig
 
 # For GHA testing
 .act/
-.actrc
\ No newline at end of file
+.actrc
+*/build/
```

Comment:
Why? We already have /build there which correctly ignores the logs.

Reply:
Sure, I can remove this. There were changes to the gitignore which meant some of my local test files were turning up, but I can put them in `/build`.